packages/code-du-travail-frontend/src/outils/common/PrevNextBar.js

```diff
@@ -46,18 +48,24 @@ PrevNextBar.defaultProps = {
 
 export { PrevNextBar };
 
-const { spacings } = theme;
+const { breakpoints, spacings } = theme;
+
+const StyledButton = styled(Button)``;
 
 const Box = styled.div`
   display: flex;
+  flex-direction: row-reverse;
```

Comment:
event I would naturally done the opposite, move the first primary action first in the dom is a good idea !

Reply:
I mostly did this for mobile. The main action is the first you reach when scrolling on mobile

Reply:
(yeah when I say the opposite is to leave previous dom order and use row revers on mobile but these way is better